agent/mission_solver/data_loader.py

```
@@ -0,0 +1,158 @@
+
+    # 收集所有匹配当前日期的条目
+    matched_entries = []
+    for entry in schedule:
+        started = _parse_date(entry.get("startedAt", ""))
+        ended = _parse_date(entry.get("endedAt", ""))
+        if started and ended and started <= target_date <= ended:
+            entry_id = entry.get("id", 0)
+            mission_count = len(entry.get("missions", []))
+            # 跳过常驻新手任务 (id=10001, 时间范围极大)
+            if entry_id == 10001 and mission_count > 100:
+                logger.debug(f"跳过常驻新手任务: id={entry_id}, {mission_count} missions")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)